calcom/cal.com
# fix: Made dynamic text for booking with only booker or multiple attendees

## What does this PR do?
Added logic for checking attendees in the booking and showing message based on that

Fixes #9239 

## Type of Change
Bug Fix (non-breaking change which fixes an issue)

## How should this be tested?

- Go to booking
- Create a booking with only 1 member 
- Now check the message
- Now create another booking with multiple members
- Now check the message
